B827EBFFFEE5E10E.json

```diff
@@ -0,0 +1,18 @@
+{ 
+	"gateway_conf": { 
+		"gateway_ID": "B827EBFFFEE5E10E", 
+		"servers": [ 
+			{ 
+				"server_address": "ttn-router-eu", 
```

Comment:
This is not a valid URL for a router, please check here: https://github.com/ttn-zh/gateway-remote-config/#how-to-contribute